src/librustc_codegen_llvm/abi.rs

```diff
@@ -507,6 +507,14 @@ impl<'tcx> FnTypeExt<'tcx> for FnType<'tcx, Ty<'tcx>> {
             }
         };
 
+        // Store the index of the last argument. This is useful
+        // for working with C compatible variadic arguments.
+        let last_arg_idx = if sig.inputs().len() > 0 {
```

Comment:
I \*think\* it's more idiomatic to use `is_empty` (clippy says so). :p

Reply:
yeah, I don't know what I was thinking 😄